tests/wpt/tests/css/CSS2/text/text-align-justify-with-overflow.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<head>
+    <title>CSS Test: Overflowing content with text-align: justify</title>
+    <link rel="author" title="Martin Robinson" href="mrobinson@igalila.com">
+    <link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+    <link rel="help" href="http://www.w3.org/TR/CSS21/text.html#indentation-prop">
+    <link ref="help" href="https://drafts.csswg.org/css-text/#text-align-property">
+    <link rel="match" href="text-align-justify-with-overflow-ref.html"/>
+    <style type="text/css">
+        div {
+            text-align-last: justify;
+            width: 0px;
+            white-space: pre-wrap;
```

Comment:
I think I added `white-space: pre-wrap` when I was testing something else, but it doesn't seem relevant now

Reply:
Okay. Removed. I wasn't sure why you added that.